source/features/link-to-gihub-io.tsx

```diff
@@ -0,0 +1,88 @@
+import React from "dom-chef";
+import elementReady from "element-ready";
+import select from "select-dom";
+import * as pageDetect from "github-url-detection";
+import LinkExternalIcon from "octicon/link-external.svg";
+
+import features from ".";
+
+function getCurrentUsername(): string {
+	return location.pathname.replace(/(^\/)/g, "").split("/")[0];
+}
+
+function getGithubIoLink(username: string): string {
+	return `${username}.github.io`;
+}
+
+async function repoTabInit(): Promise<void> {
+	const username = getCurrentUsername();
+	const githubIoLink = getGithubIoLink(username);
+	const repo = select(`a.rgh-discussion-links[href$="${githubIoLink}"]`);
+	repo?.parentElement?.appendChild(
+		<a
+			href={`https://${githubIoLink}`}
+			target="black"
+			rel="noopener noreferrer"
+		>
+			<LinkExternalIcon />
+		</a>
+	);
+	return;
+}
+
+async function githubIoRepoInit(): Promise<void> {
+	const repositoryName = pageDetect.utils.getRepoURL().split("/")[1];
+	const githubIoLink = getGithubIoLink(getCurrentUsername());
+	const link = (await elementReady(".BorderGrid-cell .octicon-link"))!;
+	if (repositoryName.endsWith(".github.io")) {
+		if (!link) {
+			const aboutTab = select(".BorderGrid-cell");
+			aboutTab?.appendChild(
+				<div className="mt-3 d-flex flex-items-center">
+					<svg
+						height="16"
+						className="octicon octicon-link flex-shrink-0 mr-2 flex-shrink-0"
+						viewBox="0 0 16 16"
+						version="1.1"
+						width="16"
+						aria-hidden="true"
+					>
+						<path
+							fill-rule="evenodd"
+							d="M7.775 3.275a.75.75 0 001.06 1.06l1.25-1.25a2 2 0 112.83 2.83l-2.5 2.5a2 2 0 01-2.83 0 .75.75 0 00-1.06 1.06 3.5 3.5 0 004.95 0l2.5-2.5a3.5 3.5 0 00-4.95-4.95l-1.25 1.25zm-4.69 9.64a2 2 0 010-2.83l2.5-2.5a2 2 0 012.83 0 .75.75 0 001.06-1.06 3.5 3.5 0 00-4.95 0l-2.5 2.5a3.5 3.5 0 004.95 4.95l1.25-1.25a.75.75 0 00-1.06-1.06l-1.25 1.25a2 2 0 01-2.83 0z"
+						></path>
+					</svg>
```

Comment:
Use 
`import OctoLinkIcon from 'octicon/octicon-link.svg'`

Then

```suggestion
	<OctoLinkIcon/>
```

Reply:
There's already `LinkExternalIcon` on the page:
```suggestion
					<LinkExternalIcon />
```

Reply:
Looks like this hasn't been resolved yet! I still see the plain `svg`.

Reply:
Done